README.md

```diff
@@ -1,51 +1,127 @@
 let bucket = new cloud.Bucket();
+```
 
-new cloud.Function(inflight (event: str): str => {
-  bucket.put("greeting.txt", "hello, world!");
+Since the Wing compiler sees the entire application, including its infrastrucrure, it can also generate least privilege IAM policies based on intent, and other cloud mechanics.
+
+```js
+bring cloud;
+
+// This code runs at compile time to provision the bucket.
+let bucket = new cloud.Bucket();
+
+// This code runs at compile time to provision the function.
+new cloud.Function(inflight () => {
+  // This code runs later, on other machines, and interacts with the provisioned bucket.
+  // The compiler knows to generate the network topology and IAM policies needed for the function to be able to put
+  // a file in the bucket. It also knows to capture the bucket at deployment time and pass the info that is needed
+  // for the function to interact with it at runtime.
+  bucket.put("hello.txt", "World!");
 });
 ```
 
+## Wing's creator
+While there are many developers [contributing](#contributors) to the project, it started with the vision of [Elad Ben-Israel], who is also the creator of the [AWS CDK].
+
+[cloud-oriented programming language]: https://docs.winglang.io/#what-is-a-cloud-oriented-language
+[distributed computing primitives]: https://docs.winglang.io/concepts/inflights
+[resource system]: https://docs.winglang.io/concepts/resources
+[other optimizations for cloud programming]: #main-features-that-make-wing-tailored-for-cloud-development
+[compiler plugins]: https://docs.winglang.io/reference/compiler-plugins
+[Elad Ben-Israel]: https://github.com/eladb
+[AWS CDK]: https://github.com/aws/aws-cdk
+[console]: https://docs.winglang.io/getting-started/console
+[Standard library]: https://docs.winglang.io/reference/wingsdk-spec
+
+## Why a language? (and not an SDK or platform)
+We believe that the cloud is a new kind of computer that requires a [new programming paradigm](https://docs.winglang.io/#what-is-a-cloud-oriented-language) to fully utilize it.
+
+While it is possible to use this new paradigm with existing languages, we believe that a language that natively supports it will take it to the next level, in a way that is impossible to accomplish with existing ones.
+
+Inflight functions and resources are a great example. We believe it is impossible to ensure this level of isolation and interaction semantics in existing languages through a library, or even through a compiler extension. We've seen some worthy efforts in projects like [Pulumi's Function Serialization](https://www.pulumi.com/docs/intro/concepts/function-serialization/), [Punchcard](https://github.com/sam-goodwin/punchcard) and [Functionless](https://functionless.org/), and in all of these cases, there are either compromises in safety (e.g. it is possible to capture mutable objects) or in the programming model (e.g. type system is too complex).
```

Comment:
Should we try to always treat `inflight` like a keyword?

```suggestion
`inflight` functions and resources are a great example. We believe it is impossible to ensure this level of isolation and interaction semantics in existing languages through a library, or even through a compiler extension. We've seen some worthy efforts in projects like [Pulumi's Function Serialization](https://www.pulumi.com/docs/intro/concepts/function-serialization/), [Punchcard](https://github.com/sam-goodwin/punchcard) and [Functionless](https://functionless.org/), and in all of these cases, there are either compromises in safety (e.g. it is possible to capture mutable objects) or in the programming model (e.g. type system is too complex).
```

Reply:
Also I'm not sure we should bother mentioning punchcard here, since functionless is a pretty direct evolution of it

Reply:
> Should we try to always treat inflight like a keyword?

I think so, yes.